test/unit/webui/app.spec.js

```diff
@@ -54,16 +54,21 @@ describe('App', () => {
   });
 
   it('isUserAlreadyLoggedIn: token already available in storage', async () => {
-
     storage.setItem('username', 'verdaccio');
-    storage.setItem('token', generateTokenWithTimeRange(24));
+    storage.setItem('token', 'TEST_TOKEN');
     const { isUserAlreadyLoggedIn } = wrapper.instance();
-
+    const result = {token: 'TEST_TOKEN', username: 'verdaccio'};
```

Comment:
`'TEST_TOKEN' is twice, so, `const`

Reply:
`const result = {token: 'TEST_TOKEN', username: 'verdaccio'};` I would suggest the local storage should return this object, is more reliable the shape match properly.